include/os/linux/spl/sys/kmem_cache.h

```diff
@@ -99,7 +99,8 @@ extern struct rw_semaphore spl_kmem_cache_sem;
 #endif
 
 #define	POINTER_IS_VALID(p)		0	/* Unimplemented */
-#define	POINTER_INVALIDATE(pp)			/* Unimplemented */
+#define	POINTER_INVALIDATE(pp)		\
+	(*(pp) = (void *)((uintptr_t)(*(pp)) | 0x1))
```

Comment:
The original purpose here was to tell the force exporting thread that the pool is ready to actually be exported, which doesn't use `POINTER_IS_VALID`.  I've changed the condition to simply use what was used to invalidate the pointer, instead of using this indirection and ugliness.